t/mojo/util.t

```diff
@@ -6,9 +6,8 @@ use lib curfile->sibling('lib')->to_string;
 use Test::More;
 use Mojo::ByteStream qw(b);
 use Mojo::DeprecationTest;
-use Sub::Util qw(subname);
```

Comment:
The changes to this test file are inconsistent, and I think this test file could actually be left alone so as to make sure that the re-exports always remain functional.